composer.json

```diff
@@ -57,7 +60,8 @@
         "classmap": [
             "app/AppKernel.php",
             "src/extensions",
-            "src/framework"
+            "src/framework",
+            "var/autoclasses"
         ],
         "psr-4": {
             "AppBundle\\": "src/AppBundle"
```

Comment:
I guess we should clean up this file. The feature specific branches should not be in there. And why are we adding rector in this context?